### PR TITLE
Enh #1: Update JS dependencies.

### PR DESCRIPTION
**Pull request by yuriimaz**
- Updated prosemirror, markdown-it, emoji, mocha and grunt dependencies.
- Added and configured EsLint module.
- Fixed building package bundle.
- Fixed adding all dependencies to the bundle.
- Fixed using serializer from prosemirror-markdown module.
- Fixed using twemoji module.
- Fixed opening upload image window.

https://github.com/humhub/humhub-prosemirror-internal/issues/1

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Changelog
 - Enh #48: Enhanced menu item visibility configuration
 - Enh #40: Emoji in headline creates new paragraph
 - Chng: Refactored history into own plugin
+- Enh #1: Update JS dependencies
```
Comment (luke-) [inline docs/CHANGELOG.md:16]: Can you please use the # id of this PR here.

I've described the Changelog Scheme here: 

https://community.humhub.com/s/contribution-core-development/wiki/Code+Style#pull-requests-%26-changelog

**Review by luke- — CHANGES_REQUESTED** [review]
@yuriimaz Thanks. Looks good. One small remark.

**Review by luke- — CHANGES_REQUESTED** [review]
@yuriimaz Thanks. Looks good. One small remark.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (yuriimaz) [reply docs/CHANGELOG.md]: Changed

**Review by luke- — APPROVED** [review]

**Pull request merged**